tests/dragonfly/acl_family_test.py

```diff
@@ -220,7 +218,7 @@ async def test_acl_deluser(df_server):
 
 
 script = """
-for i = 1, 10000 do
+for i = 1, 100000 do
```

Comment:
I slightly increased this to avoid flakes but what I run locally won't necessary reflect the gh runners since the machines we are running on the regressions tests are far more powerful than mine

Reply:
They're less powerful. The flakes I was worried about is that it setuser migth run to quickly